owlbot.py

```diff
@@ -183,3 +183,21 @@ def generate_protos(session):
     PB2_GRPC_HEADER,
     fr"{LICENSE}\n\n\g<1>\n\n\g<2>",  # add line breaks to avoid stacking replacements
 )
+
+s.replace(
+    "CONTRIBUTING.rst",
+    "master",
+    "main"
+)
+
+s.replace(
+    "docs/conf.py",
+    "master",
+    "main"
+)
+
+s.replace(
+    ".kokoro/*",
+    "master",
+    "main"
+)
```

Comment:
Did you have to use owlbot for this because we can't update the templates yet?

Reply:
Yes, I believe the plan is to update on the branches first, and then we'll update the templates. Once that's done we'll revert the changes on OwlBot.

Reply:
Another suggestion is to have a flag that switches `main`/`master` in the templates so it's a one line change. Post-migration you can just make `main` the default. 

I did something similar back when we were moving to the microgenerator.

Reply:
That sounds good @busunkim96, do you have any links for the previous PRs with those examples? Can't visualize how that'd work here :/